### PR TITLE
Migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by at-wat**

### CODE REVIEW

**Comment by at-wat** [comment]
working as expected:
- https://github.com/seqsense/aports-ros-experimental/pull/402
- https://github.com/seqsense/aports-ros-experimental/pull/401
- https://github.com/seqsense/aports-ros-experimental/pull/400

**Comment by at-wat** [comment]
@f-fl0 please take a look

File: .github/workflows/update.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: build
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+  schedule:
+    - cron:  '0 0 * * 1'  # Every monday UTC 0:00 (JST 9:00)
```
Comment (f-fl0) [inline .github/workflows/update.yaml:10]: What is necessary or this is just a good opportunity to add that feature?

**Review by f-fl0 — APPROVED** [review]
LGTM. Just left a comment about the cron schedule.

File: .github/workflows/update.yaml (hunk 1 above)
Comment (at-wat) [reply .github/workflows/update.yaml:10]: This is for triggering weekly update of aports-ros-experimental based on the official rosdistro.

**Pull request merged**